odml/property.py

```diff
@@ -129,7 +129,9 @@ def __init__(self, name=None, values=None, parent=None, unit=None,
 
         self.parent = parent
 
-        validation.Validation(self)
+        for err in validation.Validation(self).errors:
+            if err.rank == "error":
```

Comment:
Since the `ValidationError` class has a method `is_error()`, that should be used to check the rank instead of comparing against the string.  That way, if the labels ever change, we would only need to change the constant strings `LABEL_ERROR` and `LABEL_WARNING` in the validation module and everything would continue working.